xrpl4j-core/src/main/resources/definitions.json

```diff
@@ -2759,7 +2781,7 @@
     "tecKILLED": 150,
     "tecHAS_OBLIGATIONS": 151,
     "tecTOO_SOON": 152,
-    "tecHOOK_ERROR": 153,
+    "tecHOOK_REJECTED": 153,
```

Comment:
This seems unrelated to DID -- not objecting, just double-checking.

Reply:
See comment above -- this was generated by xrpl-codec-gen